sdk/core/abort-controller/review/abort-controller.api.md

```diff
@@ -21,10 +21,11 @@ export class AbortError extends Error {
 // @public
 export class AbortSignal implements AbortSignalLike {
     constructor();
-    readonly aborted: boolean;
+    get aborted(): boolean;
     addEventListener(_type: "abort", listener: (this: AbortSignalLike, ev: any) => any): void;
-    static readonly none: AbortSignal;
-    onabort?: (ev?: Event) => any;
+    dispatchEvent(_event: Event): boolean;
+    static get none(): AbortSignal;
+    onabort: ((ev?: Event) => any) | null;
```

Comment:
Did we change an undefined to null? The change in this signature doesn't seem compatible.

Reply:
Also seems like the forward-declaration of `Event` fell out from shims-public.d.ts. API Extractor doesn't yet support rolling up file-based references, so we'll have to work around it. I think core-http has a solution of some kind, but I can't remember exactly. I think @jeremymeng knows more.